src/client/mod.rs

```diff
@@ -302,40 +302,43 @@ impl ClientBuilder {
         self.intents
     }
 
-    /// Sets an event handler with multiple methods for each possible event.
-    pub fn event_handler<H: EventHandler + 'static>(mut self, event_handler: H) -> Self {
```

Comment:
I think it's better to remove the add_ prefix from `add_event_handler`, `add_event_handler_arc`, and `add_raw_event_handler` again, because it causes needless churn.

Reply:
done